tests/test_capsule/test_capsule_correctness_checks.py

```diff
@@ -0,0 +1,157 @@
+import os
+
+import pytest
+
+from umbral import pre
+from umbral.curvebn import CurveBN
+from umbral.fragments import CapsuleFrag
+from umbral.keys import UmbralPrivateKey
+from umbral.point import Point
+from umbral.pre import Capsule
+from umbral.signing import Signer
+
+
+def test_cannot_attach_cfrag_with_keys():
+    """
+    We need the proper keys to verify the correctness of CFrags
+    in order to attach them to a Capsule.
+    """
+    capsule = Capsule(point_e=Point.gen_rand(),
+                      point_v=Point.gen_rand(),
+                      bn_sig=CurveBN.gen_rand())
+
+    cfrag = CapsuleFrag(point_e1=Point.gen_rand(),
+                        point_v1=Point.gen_rand(),
+                        kfrag_id=os.urandom(10),
```

Comment:
This needs to be 32 bytes, not 10.

Reply:
Why?  What's so special about 32?  Just that it happens to be the size of a bignum represented as bytes given a curve that we use by default?  There's no harm in making it some other arbitrary number in a test.

Reply:
Because it's the way it should be (for 256-bit curves) and if there is a _right_ way to do it, then it should be done that way, imho.

Reply:
...why is that the right way?  Who says kfrag IDs have to follow curves at all?  And if they do, why 256-bit curves?  I mean, I can change it to 32, but I don't think these are good arguments.  The size is arbitrary, and I don't want to give the impression that 32 bytes is the only possible length either.

Reply:
By convention, the ID should be as long as the curve bitsize

Reply:
Yeah?  What convention?  Is that documented somewhere?

It's just a test.  There's really no reason for it to even be random in the first place.  It can just be `b"this is a kfrag ig"`